### PR TITLE
Test kitchen

### PR DESCRIPTION
**Pull request by milosgajdos83**
This adds basic tests to cover basic Jenkins setup.
It also removes content of ```plugins``` state as this will need to be reworked.

### CODE REVIEW

**Comment by milosgajdos83** [comment]
Retest this please.

**Comment by milosgajdos83** [comment]
Looks like this trick didn't work @ashb 
Either way, a link to the successful build is [here](http://jenkins.dsd.io/job/salt-formula-tester/151/console)

File: formula-requirements.txt [hunk 1]
```diff
@@ -1,2 +1,14 @@
-git@github.com:ministryofjustice/repos-formula.git
-git@github.com:ministryofjustice/nginx-formula.git
+git@github.com:ministryofjustice/nginx-formula.git==v3.2.1
+git@github.com:ministryofjustice/supervisor-formula.git==v1.1.0
```
Comment (ashb) [inline formula-requirements.txt]: Is supervisor-formula still a requirement? I seem to remember munhitsu went and removed many (though maybe not all) of our remaning cases where we pulled it in.

Comment (milosgajdos83) [inline formula-requirements.txt]: This is really out of my control as ```formula-requirements.txt``` is generated from ```metadata.yml``` by ```salt-shaker```. One of the formulas must be pulling it in as a dependency - my guess would be ```python-formula```.

**Pull request merged**